DictionaryRepo - remove insensitive terms in dependencies

reddyashish opened the pull request:
This PR is in reference to the task: https://jira.autodesk.com/browse/DYN-2851

Tested this on localhost and it looks fine. When I run "npm install", these bower_components should be downloaded locally, right?

Reviewers: @mjkkirschner @zeusongit @QilongTang

reddyashish committed:
Remove bower components

QilongTang commented:
Yes bower components should not be stored in general, I am curious if these are stored so that deployment to S3 can be made easier.. If that is the case, let's change our deployment process rather than polluting the repo

QilongTang commented:
@reddyashish Did you also check if we need to touch gitignore file so these are not accidentally committed in the future?

mjkkirschner commented:
to install bower components you may need to do `bower install`

mjkkirschner commented:
I do wonder... if we can just get rid of the bower use and move to npm for everything... that is much cleaner, but again it comes down to the plan for this repo.

reddyashish committed:
add bower_components folder to .gitignore

reddyashish commented:
@QilongTang I have added the folder to .gitignore. 
@mjkkirschner yes, we will need to run bower install requirejs, Blanket and Mocha to get those dependencies. 
Let me know what you guys think about moving to npm or sticking to bower. If we stick to bower, should this step be added to readme? or Do you guys expect it to be done during the npm install step?

reddyashish committed:
update README.md

reddyashish committed:
Update README.md

reddyashish committed:
Update README.md

reddyashish commented:
I have updated the README.md on how to install mocha, blanket and requirejs dependencies from bower. To switch to npm, it seems like it would need more changes as the dependencies would be downloaded into the node_modules folder instead. To keep it simple for now, just updated the readme file.

mjkkirschner commented:
Hi @reddyashish theres conflicts on this PR, please fix.

reddyashish committed:
Update README.md

reddyashish commented:
@mjkkirschner Resolved it.
